Task 1/Assets/Scripts/World/World.cs

```diff
@@ -0,0 +1,29 @@
+using StackApp.Blocks;
+using UnityEngine;
+
+namespace StackApp.World
+{
+    public class World : MonoBehaviour
+    {
+        private GameConfigurationDataBlock blockConfig;
+        public Block worldBlock;
+
+        public void Initialize(GameConfigurationDataBlock blockConfiguration)
+        {
+            blockConfig = blockConfiguration;
+            SpawnWorld();
+        }
+
+        private void SpawnWorld()
+        {
+            worldBlock = Instantiate(blockConfig.BlockPrefab, blockConfig.BlocksParentTransform);
+            worldBlock.CreateBlock(CalculateSize(), blockConfig.StartingBoxPosition, blockConfig.initialColor);
+        }
+
+        private Vector3 CalculateSize()//Calculate size of world box by length and width = InitialSize and height = 2*Y - (blocks height)/2
```

Comment:
Bad comment - it doesn't bring new information - method's name already tells what it does, and formula is there - in the code.
please remove